pkg/scheduler/metrics/metrics.go

```diff
@@ -95,19 +130,23 @@ var registerMetrics sync.Once
 func Register() {
 	// Register the metrics.
 	registerMetrics.Do(func() {
-		prometheus.MustRegister(E2eSchedulingLatency)
-		prometheus.MustRegister(SchedulingAlgorithmLatency)
-		prometheus.MustRegister(BindingLatency)
-
-		prometheus.MustRegister(SchedulingAlgorithmPredicateEvaluationDuration)
-		prometheus.MustRegister(SchedulingAlgorithmPriorityEvaluationDuration)
-		prometheus.MustRegister(SchedulingAlgorithmPremptionEvaluationDuration)
-		prometheus.MustRegister(PreemptionVictims)
-		prometheus.MustRegister(PreemptionAttempts)
+		for _, metric := range metricsList {
+			prometheus.MustRegister(metric)
+		}
 	})
 }
 
+// Reset resets metrics
+func Reset() {
+	SchedulingLatency.Reset()
```

Comment:
Should this reset everything? (Is this a normal prometheus pattern?)

Reply:
The problem is that simple metrics don't have reset function, only vectors do.